### DISCUSSION TITLE
GitHub申请学生一直被驳回

### DISCUSSION THREAD

**Discussion by Alonesd** [community/community#47939] [GitHub Education]
我苦战熬夜申请多次 把IP地址更改到原来主机IP地址申请还是被驳回，GitHub教育申请是不是有时间段的呀。
<img width="505" alt="截屏2023-02-19 14 27 54" src="https://user-images.githubusercontent.com/123158058/219932801-23bfaa84-a918-4341-9dc6-cdc5bba22346.png">

**Comment by samus-aran** [comment]
Hi @zhaungsix 👋 

We're happy you're here! Let's see how we can assist.

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor rather than a phone perhaps. We also accept pre-translated evidence if the text is not English. Take a photo of the translated text alongside the original copy and submit.

Lastly we take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting.
